README.md

```diff
@@ -51,7 +51,8 @@ To configure the application before installation and usage (see the next section
     ----- | -----
     `LOG_LEVEL` | The minimum level for log messages that will appear in output. `INFO` or `DEBUG` is recommended for most use cases; see [Python's logging module](https://docs.python.org/3/library/logging.html).
     `CANVAS_ACCOUNT_ID` | The Canvas instance root account ID number associated with the courses for which data will be collected.
-    `CANVAS_TERM_ID` | The Canvas instance term ID number that will be used to limit the query for Canvas courses.
+    `CANVAS_TERM_ID` | The Canvas instance term ID number that will be used to limit the query for Canvas courses. Set to 0 to just use ADD_COURSE_IDS.
```

Comment:
I realized the `0` is going to be a problem here if you want to run your jobs and `inventory.py`, since I treat `CANVAS_TERM_ID` as required. How should we deal with that? I could probably change the `inventory` code a bit.

Reply:
If it's only used for testing, it might not be a problem.

Reply:
I feel like we'll pretty much be doing full term runs with this and this is mostly being added for testing purposes.

Reply:
Sure, that make sense